hack/godep-save-dockerized.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
What's the purpose of godep-save.sh now? Is it supposed to only be used together with godep-restore.sh in verify-godeps.sh? 

I think we need to change verify-godeps.sh as well. That script is used to make sure /Godeps and /vendor are in sync. Because the output of hack/godep-save.sh is in `_output/vendor` now, so we should compare `/vendor` with `/_output/vendor` in verify-godeps.sh.